dali/base/dadfs.hpp

```diff
@@ -409,6 +411,8 @@ interface IDistributedFile: extends IInterface
  */
 interface IDistributedSuperFile: extends IDistributedFile
 {
+    virtual void attach(const char *logicalname,IUserDescriptor *user, bool link=false) = 0;     // attach to name in DFS
+
```

Comment:
See previous comment re. overloaded std. attach() definition handling logic of loading and linking subfiles.
I don't think a special variety is needed.